pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -0,0 +1,32 @@
+statement ok
+CREATE TABLE geo_table(id int primary key, geog geography(geometry, 4326), geom geometry(point, 4326))
```

Comment:
Yes, I've omitted it for now because we haven't released 20.1 yet which makes testing that hard. https://github.com/orgs/cockroachdb/projects/6#card-35995350